osbenchmark/metrics.py

```diff
@@ -2094,6 +2115,18 @@ def add_op_metrics(self, task, operation, throughput, latency, service_time, cli
             doc["meta"] = meta
         self.op_metrics.append(doc)
 
+    def add_kpi_metrics(self, task, operation, recall_at_k_stats, recall_at_1_stats, recall_time_ms_stats, error_rate, duration):
+        self.kpi_metrics.append({
+            "task": task,
+            "operation": operation,
+            "recall@k": recall_at_k_stats,
+            "recall@1":recall_at_1_stats,
+            "recall_time_ms": recall_time_ms_stats,
```

Comment:
I spoke with @VijayanB and `recall_time_ms` is not a stat we want to display. I'll remove it from the `kpi_metrics` method in the next revision.

Reply:
Can we remove from this PR?

Reply:
Sorry not sure what "this" is referring to. Do you mean remove the `add_kpi_metrics` function (and instead append directly to the `kpi_metrics` list), remove the `recall_time_ms` metric, or remove `kpi_metrics` all together and go a different direction for passing recall from metadata to the metrics store?

Reply:
sorry meant the recall_time_ms - not sure if this is needed